### PR TITLE
File Integrity Operator pod container permissions

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
This is a part of [CMP-1619](https://issues.redhat.com/browse/CMP-1619), however these changes apply only to 4.12+ branches so it is a separate PR.

This change also mirrors https://github.com/openshift/openshift-docs/pull/51481/files

Link to docs preview:
[Installing the File Integrity Operator using the CLI](http://file.rdu.redhat.com/antaylor/FIO-pod-security/security/file_integrity_operator/file-integrity-operator-installation.html#installing-file-integrity-operator-using-cli_file-integrity-operator-installation)

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

File: modules/file-integrity-operator-installing-cli.adoc [hunk 1]
```diff
@@ -27,8 +27,10 @@ kind: Namespace
 metadata:
   labels:
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged <1>
```
Comment (bergerhoffer) [inline modules/file-integrity-operator-installing-cli.adoc:30]: @sheriff-rh Actually I just realized... the PSA enforcement that was scheduled for 4.12 is getting pushed to 4.13 now. So I'm wondering if this change should actually not go in for 4.12 anymore...

@stlaz @ibihim With the restricted enforcement for PSA getting rolled back/moving to 4.13, should we be updating docs with these kinds of changes, or not? I'm aware of at least one other PR (#51481) that has added this label, and need to know if we should revert that and communicate out that we shouldn't make these changes yet for 4.12. Or if it's okay to add for 4.12?

Comment (stlaz) [reply modules/file-integrity-operator-installing-cli.adoc:30]: This kind of documentation is fine to be added even for lower versions. Properly setting namespace labels is a good thing irregardless of whether we are globally enforcing PSa or not.

**Comment by stlaz** [comment]
+1 on the LGTM

**Comment by sheriff-rh** [comment]
It's been a few days, so I'll rebase and push.

**Pull request merged**